### PR TITLE
JVMChaos: support inject fault into MySQL client

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve?

close issue https://github.com/chaos-mesh/chaos-mesh/issues/2977

(this pr is cherry-picked from https://github.com/chaos-mesh/chaos-mesh/pull/3189)

### What's changed and how it works?

Using `sql-helper` in [byteman-helper](https://github.com/chaos-mesh/byteman-helper/tree/main/BytemanHelper#sqlhelper) to analyze SQLs, and judge whether to inject fault.

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.2
  - [ ] release-2.1

### Checklist

CHANGELOG

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [x] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@STRRL PTAL again, all comments in https://github.com/chaos-mesh/chaos-mesh/pull/3189 is addressed

File: api/v1alpha1/jvmchaos_types.go [hunk 1]
```diff
@@ -59,6 +59,9 @@ const (
 	// JVMRuleDataAction represents inject fault with byteman's rule
 	// refer to https://downloads.jboss.org/byteman/4.0.14/byteman-programmers-guide.html#the-byteman-rule-language
 	JVMRuleDataAction JVMChaosAction = "ruleData"
+
+	// JVMMysqlAction represents the JVM chaos action of mysql java client fault injection
```
Comment (STRRL) [inline api/v1alpha1/jvmchaos_types.go:63]:
```suggestion
	// JVMMySQLAction represents the JVM chaos action of mysql java client fault injection
```

**Review by STRRL — COMMENTED** [review]
rest LGTM!

**Review by STRRL — APPROVED** [review]
LGTM!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**